src/Ethereum/ABI/Parameters.cpp

```diff
@@ -35,7 +35,7 @@ void ParamSet::addParams(const std::vector<std::shared_ptr<ParamBase>>& params)
 }
 
 bool ParamSet::getParam(int paramIndex, std::shared_ptr<ParamBase>& param_out) const {
-    if (paramIndex >= _params.size() || paramIndex < 0) {
+    if (static_cast<std::size_t>(paramIndex) >= _params.size() || paramIndex < 0) {
```

Comment:
pass in `size_t` instead?

Reply:
Possible, but is starts a cascading change.

Reply:
I will check for this one

Reply:
Fixed in [5afd28e](https://github.com/trustwallet/wallet-core/pull/2397/commits/5afd28e575d46c982fd8ee043e65365ca1da968f)